src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -45,11 +45,9 @@
 namespace Google\ApiCore\LongRunning\Gapic;
 
 use Google\ApiCore\ApiException;
-use Google\ApiCore\Call;
 use Google\ApiCore\GapicClientTrait;
-use Google\ApiCore\RetrySettings;
 use Google\ApiCore\RequestParamsHeaderDescriptor;
-use Google\ApiCore\Transport\TransportInterface;
+use Google\ApiCore\RetrySettings;
```

Comment:
This is a generated class, so we _probably_ don't want to make the changes here. But we are removing this soon in favor of it's own package, so it's a pretty innocuous change.